DESCRIPTION

```diff
@@ -11,10 +11,10 @@ Description: Estimate average treatment effects in panel data.
     Currently provides methods only for the case that all treated units
     adopt treatment at the same time.
 Depends:
-    R (>= 3.4)
+    R (>= 3.4),
+    mvtnorm
```

Comment:
I believe standard practice here would be to use "imports" as opposed to "depends", so we don't glob all the namespaces together. Then we can call the function as `mvtnorm::rmvnorm`.

https://stackoverflow.com/questions/8637993/better-explanation-of-when-to-use-imports-depends